setup.sh

```diff
@@ -11,10 +12,16 @@ while [ $# -gt 0 ]; do
     --lowest-ipv6-mtu=*)
       LOWEST_IPV6_MTU="${1#*=}"
       ;;
+    --handle-rst-during-fin-rcv=*)
+      HANDLE_RST_DURING_FIN_RCV="${1#*=}"
+      ;;
     *)
       printf "************************************************************\n"
       printf "* Error: Invalid argument '${1}'.\n"
-      printf "* Usage: setup.sh [--pool6=<IPv6>] [--lowest-ipv6-mtu=<mtu>]\n"
+      printf "* Usage: setup.sh [options...]\n"
+      printf "*        --pool6=<IPv6>\n"
+      printf "*        --lowest-ipv6-mtu=<mtu>\n"
+      printf "*        --handle-rst-during-fin-rcv=<boolean>\n"
```

Comment:
I would rather don't accept value for this. If this option is used, just switch `HANDLE_RST_DURING_FIN_RCV` to true.
```suggestion
      printf "*        --handle-rst-during-fin-rcv\n"
```